options/locale/locale_fa-IR.ini

```diff
@@ -1483,9 +1483,13 @@ settings.add_slack_hook_desc=درست کردن <a href="%s">سستی </a> در 
 settings.slack_token=توکن
 settings.slack_domain=دامنه
 settings.slack_channel=کانال
+settings.teamcity_host_url = Host URL
+settings.teamcity_vcs_root_id = VCS Root ID
```

Comment:
Please only change English translations as our translation system will overwrite any that aren't in its system

Reply:
✔